acceptance/tests/facts/networking_facts.rb

```diff
@@ -0,0 +1,80 @@
+test_name 'FACT-1361 - C59029 networking facts should be fully populated'
+
+#
+# This test is intended to ensure that networking facts resolves
+# as expected in AIO across supported platforms.
+#
+
+skip_test "Networking fact test is confined to AIO" if @options[:type] != 'aio'
+
+@ip_regex = /^(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$/
+
+expected_networking = {
+    "networking.dhcp"     => @ip_regex,
+    "networking.ip"       => @ip_regex,
+    "networking.ip6"      => /[a-f0-9]+:+/,
+    "networking.mac"      => /[a-f0-9]{2}:/,
+    "networking.mtu"      => /\d+/,
+    "networking.netmask"  => /\d+\.\d+\.\d+\.\d+/,
```

Comment:
That makes sense, netmask should be restricted to octets. 
Netmask type A format: 255.0.0.0
Netmask type B format: 255.255.0.0
Netmask type C format: 255.255.255.0

But in some customized netmask, any 255 in any octet can be replaced by either 128, 193, 224, 240, 248, 252, or 254.

So I will add the @netmask_regex = /^(((128|192|224|240|248|252|254)\.0\.0\.0)|(255\.(0|128|192|224|240|248|252|254)\.0\.0)|(255\.255\.(0|128|192|224|240|248|252|254)\.0)|(255\.255\.255\.(0|128|192|224|240|248|252|254)))$/ in the next push